tests/integrations_plugin_wordpress_seo/Events/Integrations/Plugins/WordPress_SEO/ProviderTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\WordPress_SEO;
+
+
+/**
+ * Class Provider_Test
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\WordPress_SEO
+ */
+class ProviderTest extends \Codeception\TestCase\WPTestCase {
+
+	/**
+	 * @return Provider
+	 */
+	private function make_instance() {
+		return new Provider( tribe() );
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_be_instantiatable() {
+		$sut = $this->make_instance();
+
+		$this->assertInstanceOf( Provider::class, $sut );
+	}
+
+	/**
+	 * @test
+	 */
+	public function because_yoast_is_active_it_should_load() {
```

Comment:
Thought: You can invert this test and use `uopz_set_return` on load_conditionals.
You should also test the hooks.